src/docs/perf/power.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Preserving your battery
+---
+
```

Comment:
@terrylucas @kenzieschmoll @DaveShuckerow 
Currently the best we can do in DevTools would be to visualize what fraction of the time the app was keeping the dart isolates active.
IOS and Android both provide some battery usage charts in their debugging tools.